exec/memory/pom.xml

```diff
@@ -34,4 +34,5 @@
   <modules>
     <module>base</module>
   </modules>
+
```

Comment:
Avoid formatting changes

Reply:
Removed the new empty line (introduced by accident).